src/utils/index.ts

```diff
@@ -3,12 +3,12 @@
 // eslint-disable-next-line no-nested-ternary
 export const ROOT_URL = process.env.TARGET_ENV === 'prod'
   ? 'https://betmate-backend-prod.herokuapp.com'
-  : process.env.TARGET_ENV === 'dev'
+  : process.env.TARGET_ENV === 'dev' || true
     ? 'https://betmate-backend-dev.herokuapp.com'
     : 'http://localhost:9090';
 
 // Auth token name for storage and transmission to backend
 export const authTokenName = 'authToken';
 
 // Number of ms before an axios request times out
-export const requestTimeout = 1000;
+export const requestTimeout = 3000;
```

Comment:
This hopefully resolves some of the timeout issues

Reply:
an axios retry package might also be helpful to automatically retry requests a set number of times when they timeout

Reply:
Yeah that may be worth looking into